src/main/java/com/conveyal/gtfs/api/graphql/RouteFetcher.java

```diff
@@ -44,6 +45,27 @@
         return routes;
     }
 
+    public static List<WrappedGTFSEntity<Route>> fromStop(DataFetchingEnvironment environment) {
+        WrappedGTFSEntity<Stop> stop = (WrappedGTFSEntity<Stop>) environment.getSource();
+        List<String> routeIds = environment.getArgument("route_id");
```

Comment:
I'm kind of inclined to keep it.  Even with the top-level stopTimes traversal down to routes, it seems like a useful shorthand for getting at stop-route pairs.